core/drivers/pmd.cc

```diff
@@ -50,6 +50,16 @@ static const rte_eth_conf default_eth_conf(const rte_eth_dev_info &dev_info) {
                 dev_info.flow_type_rss_offloads,
   };
 
+  if (nb_rxq < 2)
```

Comment:
It can be simply like this in Line 43?
` ret.rxmode.mq_mode = (nb_rxq > 1) ? ETH_MQ_RX_RSS : ETH_MQ_RX_NONE;`

(I believe `rss_hf` is just ignored unless RSS is used)

Reply:
(also please run `clang-format`)

Reply:
Thanks, just pushed the corrections after tests. Changing the mode was sufficient to get the port attachment to work again. I also ran clang-format.